packages/doc/docs/extensions/customize-extension.mdx

```diff
@@ -0,0 +1,282 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Customize Extension
+
+Beside use the published extensions, you could also customized extensions by using our base extension components .
```

Comment:
```suggestion
Besides using the published extensions, you could also customize extensions by using our base extension components.
```
We should use "besides" for "In addition to" ~

Reply:
I have been fixed the words, thanks !